src/NHibernate/Tuple/Entity/PocoEntityTuplizer.cs

```diff
@@ -306,6 +307,16 @@ public override bool IsLifecycleImplementor
 
 		public override void SetPropertyValue(object entity, int i, object value)
 		{
+			//If there is no property setter we need to manually intercept value for proper lazy property handling
```

Comment:
Nit for readability: needs space between comment chars and comment.
```suggestion
			// If there is no property setter we need to manually intercept value for proper lazy property handling.
```